src/main/java/mate/controller/IndexController.java

```diff
@@ -0,0 +1,16 @@
+package mate.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class IndexController extends HttpServlet {
+    @Override
+    public void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        req.getRequestDispatcher("/webinf/views/index.jsp").forward(req, resp);
+        super.doGet(req, resp);
+    }
+}
```

Comment:
from a checklist: Always add one empty line to the end of the file.

Reply:
not sure, but I think they are just hidden here